vagrant-kubernetes-istio/RunTestOnHost/README.md

```diff
@@ -67,6 +67,22 @@ cd $ISTIO/istio
 make e2e_simple E2E_ARGS="--use_local_cluster"
 ```
 
+### 3. Debug with Delve
+You can try debugging Istio with debugger tool. vm_setup.sh already installs Delve into VM for us. To use Delve, we need process id of the binary we want to debug.
+For example, if you want to debug a test, run that test in your host/vm.
+```bash
+# In the VM/Host
+make e2e_simple E2E_ARGS="--use_local_cluster --skip_cleanup"
+```
+Now, find the pid of the process say pilot-discovery that you want to debug. If you run test in VM, you need to ssh to VM first.
```

Comment:
Run test in Host instead on run test in VM
also, can we give command vagrant ssh here so that folks know how to ssh into VM..